src/views/Login.vue

```diff
@@ -14,7 +14,7 @@
 </template>
 
 <script>
-import UserProperties from "@/components/UserProperties.vue";
+import UserProperties from "@/components/FunctionalComponents/UserProperties.vue";

```

Comment:
`FunctionalComponents` - the name is so vague, it's not useful. Like if someone was thinking where `UserProperties` is, why would they look in `FunctionalComponents`?

Reply:
Agreed
My logic - some components don't have a visual view, and are just used for some functionality
What do you suggest here?

Reply:
`Utils` is the most commonly used name, maybe that?

But basically, this will just collect things that don't belong in other places. Might as well be in the root folder. Maybe `Utils` is good 🤔

Reply:
Okay